src/context-selection/attribute-option-combo-selector-bar-item/use-should-component-render-null.js

```diff
@@ -13,6 +13,7 @@ export default function useShouldComponentRenderNull(
     const [orgUnitId] = useOrgUnitId()
 
     return (
+        categoryCombination?.isDefault ||
```

Comment:
check in lines 23-24:  `(!categoryCombination.isDefault && !categoryCombination.categories.length)` seems like it can now be simplified to `!categoryCombination.categories.length`